app/src/main/kotlin/com/wire/android/ui/home/conversations/usecase/GetMessagesForConversationUseCase.kt

```diff
@@ -23,8 +24,13 @@ class GetMessagesForConversationUseCase
 
     @OptIn(ExperimentalCoroutinesApi::class)
     suspend operator fun invoke(conversationId: ConversationId): Flow<List<UIMessage>> =
-        getMessages(conversationId).flatMapLatest { messages ->
-            observeMemberDetailsByIds(messageMapper.memberIdList(messages))
+        getMessages(conversationId)
```

Comment:
why do we have a usecase in Reloaded project, not in Kalium????

Reply:
This was already there I thin kthe reason for that is that we expose here UIMessage which is a UI model